### PR TITLE
Test changes for Raspbian execution

### PR DESCRIPTION
**Pull request by JamesWTruher**
# PR Summary

A few test fixes so tests run cleaner on raspbian

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1 [hunk 1]
```diff
@@ -63,12 +63,17 @@ Describe "Test-Connection" -tags "CI" {
```
Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1]: @JamesWTruher, your last commit had 2 failures in `PowerShell-CI-linux`
DSC MOF Compilation.Should be able to compile a MOF from a basic configuration on Windows
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 File f1
+                 ~~~~
Undefined DSC resource 'File'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 178
178:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from a configuration with multiple resources on Windows
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 File f1
+                 ~~~~
Undefined DSC resource 'File'. Use Import-DSCResource to import the resource.

At line:10 char:17
+                 Script s1
+                 ~~~~~~
Undefined DSC resource 'Script'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 198
198:         [Scriptblock]::Create(@"

```

**Comment by adityapatwardhan** [comment]
@JamesWTruher  please have a look at the failures

File: test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1 (hunk 1 above)
Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1]: @JamesWTruher, your last commit had 2 failures in `PowerShell-CI-linux`
DSC MOF Compilation.Should be able to compile a MOF from a basic configuration on Windows
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 File f1
+                 ~~~~
Undefined DSC resource 'File'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 182
182:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from a configuration with multiple resources on Windows
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 File f1
+                 ~~~~
Undefined DSC resource 'File'. Use Import-DSCResource to import the resource.

At line:10 char:17
+                 Script s1
+                 ~~~~~~
Undefined DSC resource 'Script'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 202
202:         [Scriptblock]::Create(@"

```

Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1]: @JamesWTruher, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/6.0.md.https://msdn.microsoft.com/powershell/reference/6/about/about_jobs should work
```powershell
Failed to complete request to "https://msdn.microsoft.com/powershell/reference/6/about/about_jobs". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1]: @JamesWTruher, your last commit had 3 failures in `PowerShell-CI-windows`
DSC MOF Compilation.Should be able to compile a MOF from a basic configuration
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 nxFile f1
+                 ~~~~~~
Undefined DSC resource 'nxFile'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\MOF-Compilation.Tests.ps1: line 29
29:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from another basic configuration
```powershell
Exception calling "Create" with "1" argument(s): "At line:10 char:28
+                     User = "root";
+                            ~~~~~~
Unexpected token '"root"' in expression or statement.

At line:5 char:17
+                 nxScript f1
+                 ~~~~~~~~
Undefined DSC resource 'nxScript'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\MOF-Compilation.Tests.ps1: line 48
48:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from a complex configuration
```powershell
Expected no exception to be thrown, but an exception "Errors occurred while processing configuration 'WordPressServer'." was thrown from D:\a\1\s\src\powershell-win-core\bin\release\netcoreapp3.1\win7-x64\publish\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.psm1:3745 char:5
    +     throw $ErrorRecord
    +     ~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\MOF-Compilation.Tests.ps1: line 173
173: "@) | Should -Not -Throw
```

**Pull request closed without merging**